api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -39,9 +39,20 @@
  * methods will be written to the database at startup if they don't exist yet.
  */
 public final class OpenmrsConstants {
-
-	private OpenmrsConstants() {
-	}
+	/**
+	 * @see OpenmrsUtil#getApplicationDataDirectory()
```

Comment:
why did you move these `APPLICATION_DATA` constants up? I don't see the reason. Can you please undo this change.

Reply:
@teleivo ok I will make the necessary changes.Thanks